package util.exception;

/**
 * Represent an error occurs when the user tries to find the task that is not on the list
 */
public class NoItemException extends Exception{
}
